src/components/Organizations/Projects/index.js

```diff
@@ -34,14 +35,20 @@ const OrgProjects = ({ projects = [], organizationId, organizationName, refresh,
       key: 'name',
       render: project => {
         return (
-          <ProjectGroupLink
-            projectGroupSlug={project.name}
-            organizationSlug={organizationId}
-            organizationName={organizationName}
-            key={project.id}
-          >
-            {project.name}
-          </ProjectGroupLink>
+          <>
+            <ProjectGroupLink
+              projectGroupSlug={project.name}
+              organizationSlug={organizationId}
+              organizationName={organizationName}
+              key={project.id}
+            >
+              {project.name}
+            </ProjectGroupLink>
+
+            <ProjectLink projectSlug={project.name} key={project.id}>
+              <ProjectRoute>ROUTE</ProjectRoute>
```

Comment:
Technically these aren't project routes - maybe ProjectDashboard would be better - and maybe opening in a new tab by default?
```suggestion
              <ProjectRoute>View in Dashboard</ProjectRoute>
```